content/departments/cloud/technical-docs/custom_domain.md

```diff
@@ -92,3 +92,139 @@ It is worth nothing it does not make us any less secure. Our origin server still
 No. Certificate issuance and renewal is DNS-based, so we can ask customers to create the DNS records at any time. Meanwhile, the previous `*.sourcegraph.com` domain will continue to work as it is. Once the DNS records are created and confirmed by Cloudflare, the customer can start accessing the Sourcegraph instance via the custom domain.
 
 In fact, the `*.sourcegraph.com` domain will continue to work even after custom domain is switched over. However, due to the application limitation, we only recommend customers to use custom domain to access the Sourcegraph instance once it has been turned on.
+
+## Operation
+
+### Migrate current instances as a custom domain of the new Cloud domain
+
+As of 2023-05-03, new Cloud instances are provisioned with new domains of `sourcegraphcloud.com` or `sgdev.dev`. This seciton will walk you through the process of migrating `customer.sourcegraph.com` as a custom domain of `customer.sourcegraphcloud.com`.
+
+In the following example, we will migrate `sourcegraph.sourcegraph.com` as a custom domain of the managed `sourcegraph.sourcegraphcloud.com`.
+
+#### Prereq
+
+Install the Cloudflare CLI,
+
+```sh
+go install github.com/cloudflare/cloudflare-go/cmd/flarectl@latest
+```
+
+Configure env var
+
+```sh
+export CF_API_KEY="$(op item get 'https://start.1password.com/open/i?a=HEDEDSLHPBFGRBTKAKJWE23XX4&v=qxzajcksgc3givogl3r6qjbimu&i=oeirz3a43aoeuk2nkruemwlnw4&h=team-sourcegraph.1password.com' --fields label=CLOUDFLARE_USER_API_KEY)"
```

Comment:
I think this will do funky things if you don't login to 1pass first, might want to add as a prereq

Reply:
this is the prereq 😂

I added a note below to `echo` each env var to double check

Reply:
Adding a link to how to set up `op` would be great.